neuromation/cli/job.py

```diff
@@ -739,6 +751,14 @@ def format_fail(job: str, reason: Exception) -> str:
     help="File with environment variables to pass",
     secure=True,
 )
+@option(
+    "-l",
+    "--limit",
+    metavar="MINUTES",
+    type=int,
+    help="Optional job's run-time limit in minutes (set '0' to disable)",
+    default=None,
+)
```

Comment:
I am not sure if the names are good, let's change in case of any objection:
1. `-l/--limit` or `--timeout`? or `--max-run-time`
2. is `0` good to disable the feature, or should we have a separate flag `--no-limit`?
3. is limit in minutes good? note, the server expects the value in minutes, so there's no reason to have user-faced seconds.

Reply:
`--no-limit` is not needed